modules/geometry-calculation.js

```diff
@@ -1,38 +1,53 @@
-const Line = require('./ecs6-class/line')
+const Line = require('./ecs6-class/line');
+const Point = require('./ecs6-class/point');
 
 const calculateDistance = (point1, point2) => {
-    let distanceX = (point2.x - point1.x) ** 2;
-    let distanceY = (point2.y - point2.y) ** 2;
-    const distance = Math.sqrt(distanceX + distanceY);
-    return distance;
+    if (point1 instanceof Point || point2 instanceof Point) {
+        let distanceX = (point2.x - point1.x) ** 2;
+        let distanceY = (point2.y - point1.y) ** 2;
+        const distance = Math.sqrt(distanceX + distanceY);
+        return distance;
+    }
+    else
+        throw Error("this point is not a Point")
```

Comment:
which parameter is not a point
and it will be good if I would be noticed if a parameter in undefined